Remove use_c10_dispatcher: unboxed_only flags

smessmer opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #28186 Remove c10->ATen registration forwarding
* **#28187 Remove use_c10_dispatcher: unboxed_only flags**
* #28185 Remove codegen for use_c10_dispatcher: no
* #27978 Migrate remaining ops to the c10 dispatcher
* #28208 Move complex extension test to c10
* #28184 Fix codegen for out operators
* #28183 Allow migrating factory methods to c10
* #28182 Fix overload names
* #28181 Add unsupported types to schema type parser

This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

smessmer committed:
Remove use_c10_dispatcher: unboxed_only flags

This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

dzhulgakov commented:
Can you just merge it with #27978 please? It introduces a lot of native_functions.yaml changes and might have issues with rebase for other people.

smessmer committed:
Update on "Remove use_c10_dispatcher: unboxed_only flags"


This is the default now, so no need to specify it for each op in native_functions.yaml

Differential Revision: [D17969295](https://our.internmc.facebook.com/intern/diff/D17969295/)

[ghstack-poisoned]

smessmer commented:
merged into https://github.com/pytorch/pytorch/pull/27978
